content/devdocs/references/p2p_networking.md

```diff
@@ -315,7 +511,17 @@ message header has been omitted and the actual IP address has been
 replaced with a [RFC5737][] reserved IP address.)
 
 {{ <highlight text> }}
-GIVP message hex dump
+GivePeersMessage:
+```
+1a 00 00 00 00 00 00 00 03 00 00 00 5d 87 b2 76 
```

Comment:
No need to add full message . It's too much , especially if messages are long. Annotated message would be enough. Please remove